tests/test_grad.py

```diff
@@ -36,15 +34,14 @@ def test_fwd_j1(skip_hps):
     gradcheck(tf.FWD_J1.apply, input, eps=eps, atol=atol)
 
 
-@pytest.mark.parametrize("normalize", [True, False])
 @pytest.mark.parametrize("skip_hps", [[0, 1], [1, 0]])
-def test_fwd_j2(skip_hps, normalize):
```

Comment:
why remove the test for non-default value of `normalize` here? is `fwd` not differentiable anymore?

Reply:
ok, we have an answer. it's because `normalize` was removed from `FWD_J2PLUS`